app/src/components/tables/BlockTable/BlocksTable.test.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { Table } from 'src/components';
+import {
+  getMockBlocks,
+  mockBlocksTableColumns,
+  getMockBlocksTableFooter,
+  getMockBlocksTableHeader,
+} from 'src/mocks/mock-blocks-table';
+import { formatDate, truncateHash } from 'src/utils';
+import { render } from '../../../test-utils';
+
+const blocks = getMockBlocks();
+const mockBlocksTableFooter = getMockBlocksTableFooter();
+const mockBlocksTableHeader = getMockBlocksTableHeader();
+
+describe('BlocksTable', () => {
+  it('should render the BlocksTable', () => {
+    render(
+      <Table
+        tableBodyLoading={false}
+        header={mockBlocksTableHeader}
+        columns={mockBlocksTableColumns}
+        isLastPage
+        data={blocks}
+        footer={mockBlocksTableFooter}
+      />,
+    );
+    const blocksBaseTable = screen.getByTestId('base-table');
+    const blocksTableFooter = screen.getByTestId('blocks-table-footer');
+    const blocksTableHeader = screen.getByTestId('blocks-table-header');
+
+    expect(blocksBaseTable).toBeInTheDocument();
+    expect(blocksTableFooter).toBeInTheDocument();
+    expect(blocksTableHeader).toBeInTheDocument();
+  });
+
+  it('should render Timestamp', () => {
+    render(
+      <Table
+        header={mockBlocksTableHeader}
+        columns={mockBlocksTableColumns}
+        isLastPage
+        data={blocks}
+        footer={mockBlocksTableFooter}
+      />,
+    );
+    const timestamp = screen.getAllByTestId('timestamp');
+    expect(timestamp).toHaveLength(10);
+    expect(timestamp[0]).toHaveTextContent(
+      formatDate(new Date('2023-06-23T18:14:15+00:00')),
+    );
+  });
+
+  it('should render block height', () => {
+    render(
+      <Table
+        header={mockBlocksTableHeader}
+        columns={mockBlocksTableColumns}
+        isLastPage
+        data={blocks}
+        footer={mockBlocksTableFooter}
+      />,
+    );
+    const blockHeight = screen.getAllByTestId('block-height');
+    expect(blockHeight).toHaveLength(10);
+    expect(blockHeight[0]).toHaveTextContent('1');
```

Comment:
See above. If we're trying to compare what should be in the DOM to our mock data that's what we should be accessing instead of hardcoding strings in assertions